dashboard/app/views/pd/application/teacher1920_application_mailer/confirmation.html.haml

```diff
@@ -44,7 +44,7 @@
   Thanks again for applying!
 
 %p
-  Anthonette Peña
+  Liz Gauthier
```

Comment:
Adjusts the signature on three emails:

- `confirmation__with_partner`
- `confirmation__with_partner_no_contact`
- `confirmation__without_partner`